engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -267,16 +268,20 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
         workflowLogger.info(s"Parsing workflow as ${validFactory.languageName} ${validFactory.languageVersionName}")
         pushLanguageToMetadata(validFactory.languageName, validFactory.languageVersionName)
       }
+      
+      factory
```

Comment:
preexisting, but the "there is no valid factory" path seems even more interesting to log 🙂

Reply:
It looks like there's a default factory that we always fall back to. Also `factory` here is an `ErrorOr` and the errors will be logged at some point later if it ends up as an `Invalid`